src/GCRealTimeMon/Utilities/CommandLineUtilities.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Collections.Generic;
+using static realmon.Program;
+
+namespace realmon.Utilities
+{
+    public static class CommandLineUtilities
+    {
+        /// <summary>
+        /// Sentinel path that'll be inserted if -c has no specified value.
+        /// This value will be matched in the case of deciding which configuration to use.
+        /// This is to facilitate the behavior to overwrite the default config based on the inputs of the prompts.
+        /// </summary>
+        public const string SentinelPath = "__SENTINEL_PATH__";
+        public const string RequiredCommandNotProvided = @"
+Required Command was not provided. The monitoring begins only when a process name via -n or a process id via -p is supplied.
+Examples:
+    - GCRealTimeMon.exe -n devenv / dotnet-gcmon -n devenv
```

Comment:
You need to figure out what is the current executing assembly to make the difference between GCRealTimeMon and dotnet-gcmon (see https://github.com/Maoni0/realmon/blob/main/src/GCRealTimeMon/Program.cs#L183).
This applies to other hardcoded "GCRealTimeMon" in that file